added xiao_esp32c3 conf and overlay files to allow proper compilation…

rabarar opened the pull request:
… for the xiao_esp32c3 board

rabarar committed:
samples: net: wifi: boards: added missing  files for xiao_esp32c3

The conf and overlay files to properly compile the wifi samples were
omitted for the Xiao_esp32c3 board. I've added those two files and
confirmed that the sample properly compiles and runs.

Signed-off-by: Robert Baruch <robertbaruch@mac.com>

rabarar commented:
sample: net: wifi: boards: added missing .conf and .overlay files for xiao_esp32c3

The conf and overlay files to properly compile the wifi samples were omitted for the Xiao_esp32c3 board. I've added those two files and confirmed that the sample properly compiles and runs.

Signed-off-by: Rob Baruch <robertbaruch@mac.com>

kartben commented:
> sample: net: wifi: boards: added missing .conf and .overlay files for xiao_esp32c3
> 
> The conf and overlay files to properly compile the wifi samples were omitted for the Xiao_esp32c3 board. I've added those two files and confirmed that the sample properly compiles and runs.
> 
> Signed-off-by: Rob Baruch [robertbaruch@mac.com](mailto:robertbaruch@mac.com)

If you actually amend your commit to be that, it should OK! Note that your committee name also need to be update to be "Robert Baruch" not "rabarar". 
Also you should not have to close the PR while working through this, just force push your updates. Thanks!

rabarar commented:
samples: net: wifi: boards: added missing .conf and .overlay files for xiao_esp32c3

The conf and overlay files to properly compile the wifi samples were omitted for the Xiao_esp32c3 board. I've added those two files and confirmed that the sample properly compiles and runs.

Signed-off-by: Robert Baruch [robertbaruch@mac.com](mailto:robertbaruch@mac.com)

rabarar commented:
I'm missing it - if someone what's to give me the copy of how they'd like this to appear to allow it to pass - feel free - I've tried and I'm not getting past the Compliance Check... Best, Rob

kartben commented:
You need to actually update your actual commit, as that's what ends up in the code repository eventually, not the PR contents :)

```
git commit --amend --author "Robert Baruch <robertbaruch@mac.com>" --signoff

... Update the commit message title and description 

git push --force
```

rabarar commented:
Ah - thank you!


> On Dec 5, 2023, at 3:54 PM, Benjamin Cabé ***@***.***> wrote:
> 
> 
> You need to actually update your actual commit, as that's what ends up in the code repository eventually, not the PR contents :)
> 
> git commit --amend --author "Robert Baruch ***@***.***>" --signoff
> 
> ... Update the commit message title and description 
> 
> git push --force
> —
> Reply to this email directly, view it on GitHub <https://github.com/zephyrproject-rtos/zephyr/pull/66190#issuecomment-1841596799>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/ABB2J6DVR2BTRUEJHZCW7LLYH6CZXAVCNFSM6AAAAABAIHUGVKVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTQNBRGU4TMNZZHE>.
> You are receiving this because you modified the open/close state.
>

rabarar commented:
I give up - still failing …


> On Dec 5, 2023, at 3:54 PM, Benjamin Cabé ***@***.***> wrote:
> 
> 
> You need to actually update your actual commit, as that's what ends up in the code repository eventually, not the PR contents :)
> 
> git commit --amend --author "Robert Baruch ***@***.***>" --signoff
> 
> ... Update the commit message title and description 
> 
> git push --force
> —
> Reply to this email directly, view it on GitHub <https://github.com/zephyrproject-rtos/zephyr/pull/66190#issuecomment-1841596799>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/ABB2J6DVR2BTRUEJHZCW7LLYH6CZXAVCNFSM6AAAAABAIHUGVKVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTQNBRGU4TMNZZHE>.
> You are receiving this because you modified the open/close state.
>

kartben commented:
Almost there it seems
The entirety of the commit message should literally read.

```
samples: net: wifi: add missing .conf and .overlay files for xiao_esp32c3

The conf and overlay files to properly compile the wifi samples were
omitted for the Xiao_esp32c3 board. I've added those two files and confirmed
that the sample properly compiles and runs.  

Signed-off-by: Robert Baruch
robertbaruch@mac.com
```

Note that I've made sure to wrap lines at 72 characters as per the guidelines
Going through `git commit --amend` with this text as a full replacement for what you currently have should do the trick...

rabarar commented:
Copy and pasted -> still failed…. lol the PR is harder than the fix…On Dec 5, 2023, at 4:13 PM, Benjamin Cabé ***@***.***> wrote:﻿
Almost there it seems
The entirety of the commit message should literally read.
samples: net: wifi: add missing .conf and .overlay files for xiao_esp32c3

The conf and overlay files to properly compile the wifi samples were
omitted for the Xiao_esp32c3 board. I've added those two files and confirmed
that the sample properly compiles and runs.  

Signed-off-by: Robert Baruch
***@***.***

Note that I've made sure to wrap lines at 72 characters as per the guidelines
Going through git commit --amend with this text as a full replacement for what you currently have should do the trick...

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you modified the open/close state.Message ID: ***@***.***>

kartben commented:
I am not sure what tool/editor your using for Git but fwiw your commit message actually looks like this: https://github.com/rabarar/zephyr/commit/934ba0a2fa253a9aedc0e4929e4e152732db3dfd , i.e. your successive changes have all been appended to one another?

rabarar commented:
I’m using vi and the command line git invocation to amend the comment…On Dec 5, 2023, at 5:09 PM, Benjamin Cabé ***@***.***> wrote:﻿
I am not sure what tool/editor your using for Git but fwiw your commit message actually looks like this: ***@***.*** , i.e. your successive changes have all been appended to one another?

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you modified the open/close state.Message ID: ***@***.***>

rabarar commented:
I used a different terminal and edited the file… I saw the same - not sure why the terminal wasn’t showing the file properly…Did this one pass?On Dec 5, 2023, at 5:09 PM, Benjamin Cabé ***@***.***> wrote:﻿
I am not sure what tool/editor your using for Git but fwiw your commit message actually looks like this: ***@***.*** , i.e. your successive changes have all been appended to one another?

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you modified the open/close state.Message ID: ***@***.***>

kartben commented on samples/net/wifi/boards/xiao_esp32c3.conf:
```diff
@@ -0,0 +1,11 @@
+CONFIG_WIFI=y
```
@rabarar since the commit "title" is also too long, as per the compliance check report, I'd advice to just use the below verbatim, as this should solve both issues:

```
samples: net: wifi: add missing .conf and .overlay files for xiao_esp32c3

The conf and overlay files to properly compile the wifi samples were
omitted for the Xiao_esp32c3 board. I've added those two files and confirmed
that the sample properly compiles and runs.  

Signed-off-by: Robert Baruch <robertbaruch@mac.com>
```

I know this may sound overly pedantic, but with 10s of thousands of commit in the repo, we are trying to follow some basic rules to make digging into the history easier etc. And sign-off is also important to track code provenance and Developer Certificate of Origin. Thank you for your patience!
